pkg/resources/spark/spark.go

```diff
@@ -21,24 +21,36 @@ const (
 	ApplicationName = "spark"
 )
 
+func FrameworkConfigMapName(instance string, comp Component) string {
+	return InstanceObjectName(fmt.Sprintf("%s-%s", instance, "framework"), comp)
+}
+
+func KeyTabConfigMapName(instance string, comp Component) string {
+	return InstanceObjectName(fmt.Sprintf("%s-%s", instance, "keytab"), comp)
+}
+
 // InstanceObjectName returns the name that will be used to create most owned cluster resources.
 func InstanceObjectName(instance string, comp Component) string {
 	if comp == ComponentNone {
-		return fmt.Sprintf("%s-%s", instance, ApplicationName)
+		return instance
```

Comment:
Why don't you want to include "spark" in the instance object name for all resources? If I create a Spark and Dask cluster called "foo" and "bar", respectively, then having resources named "foo-spark-*" and "bar-desk-*" is very descriptive.

If a consumer is adding "spark" to the name, then I think they should stop, no?